_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -150,29 +149,6 @@ wget -q https://deb.theforeman.org/pubkey.gpg -O- | sudo apt-key add -
 {% endhighlight %}
 </div>
 
-<div class="quickstart_os quickstart_os_ubuntu1804">
-  <p>
-    Using Puppet 6.x is recommended, which is available from the Puppet Labs repository.
-
-    To use Puppet 6.x with Puppet Agent and Puppet server:
-  </p>
-
-{% highlight bash %}
-sudo apt-get -y install ca-certificates
-cd /tmp && wget https://apt.puppet.com/puppet6-release-bionic.deb
-sudo dpkg -i /tmp/puppet6-release-bionic.deb
-{% endhighlight %}
-
-  <p>Enable the Foreman repo:</p>
-
-{% highlight bash %}
-echo "deb http://deb.theforeman.org/ bionic {{page.version}}" | sudo tee /etc/apt/sources.list.d/foreman.list
-echo "deb http://deb.theforeman.org/ plugins {{page.version}}" | sudo tee -a /etc/apt/sources.list.d/foreman.list
-sudo apt-get -y install ca-certificates
-wget -q https://deb.theforeman.org/pubkey.gpg -O- | sudo apt-key add -
-{% endhighlight %}
-</div>
-
 <div class="quickstart_os quickstart_os_ubuntu2004">
   <p>
     Using Puppet 6.x is recommended, which is available from the Puppet Labs repository.
```

Comment:
unrelated, but I think we should switch those paragraphs to Puppet 7?

Reply:
Yes, that is indeed also something I noticed.